lib/models/Coffee.js

```diff
@@ -0,0 +1,83 @@
+const pool = require('../utils/pool');
+
+module.exports = class Coffee {
+  id;
+  name;
+  roast;
+
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.roast = row.roast;
+  }
+  static async insert({ name, roast }) {
+    const { rows } = await pool.query(
+      `
+        INSERT INTO
+            coffees (name, roast)
+        VALUES
+        ($1, $2)
+        RETURNING
+        *
+        `,
+      [name, roast]
+    );
```

Comment:
We should add early returns here in the case of nothing being returned from the query so that we don't end up with an error. Something like `if(!rows[0] return null`